lib/sms_api.rb

```diff
@@ -19,19 +19,23 @@ def authorization
   def sms_deliver(phone, code)
     return false unless end_point_available?
 
-    response = client.call(:enviar_sms_simples, message: request(phone, code))
+    response = client.call(:sms_text_submit, message: request(phone, code))
     success?(response)
   end
 
   def request(phone, code)
-    { autorizacion:  authorization,
-      destinatarios: { destinatario: phone },
-      texto_mensaje: "Clave para verificarte: #{code}. Gobierno Abierto",
-      solicita_notificacion: "All" }
+    {
+      Authorization: authorization,
+      DeliveryReport: "All",
```

Comment:
Since we're using `convert_request_keys_to: :camelcase`, do we have to use camel case here?